src/components/switch.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { css, Input } from 'theme-ui';
+
+const defaultVariants = {
+  sm: {
+    width: '30px',
+    height: '15px',
+    border: '#ddd',
+    bg: '#ddd',
+    color: '#fff',
+  },
+  md: {
+    width: '38px',
+    height: '19px',
+    border: '#ddd',
+    bg: '#ddd',
+    color: '#fff',
+  },
+  lg: {
+    width: '44px',
+    height: '22px',
+    border: '#ddd',
+    bg: '#ddd',
+    color: '#fff',
+  },
```

Comment:
@a-saran 

- [ ] Why all the color code values are hardcoded here? Shouldn't this be referred from the theme?